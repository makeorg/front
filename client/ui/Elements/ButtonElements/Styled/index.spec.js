import React from 'react';
import renderer from 'react-test-renderer';
import { RedButtonStyle, IconWrapperStyle, GreyButtonStyle } from '.';

describe('ButtonElements/Styled', () => {
  describe('RedButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<RedButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('IconWrapperStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<IconWrapperStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });

  describe('GreyButtonStyle', () => {
    it('snapshot by default', () => {
      const component = renderer.create(<GreyButtonStyle />).toJSON();
      expect(component).toMatchSnapshot();
    });
  });
});
