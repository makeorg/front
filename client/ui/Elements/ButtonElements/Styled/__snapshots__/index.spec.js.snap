// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonElements/Styled ActiveButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
}

.c0 svg {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg {
  fill: rgb(74,74,74);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled ButtonSmallWrapperStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px 0;
  max-width: 410px;
}

@media (min-width:767px) {
  .c0 {
    -webkit-flex-flow: row;
    -ms-flex-flow: row;
    flex-flow: row;
    margin: 20px 0;
  }
}

<div
  className="3 c0"
/>
`;

exports[`ButtonElements/Styled ButtonsWrapperStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 10px 0;
}

@media (min-width:767px) {
  .c0 {
    -webkit-flex-flow: row;
    -ms-flex-flow: row;
    flex-flow: row;
    margin: 20px 0;
  }
}

<div
  className="c0"
/>
`;

exports[`ButtonElements/Styled CloseButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  border: none;
  background-color: transparent;
  position: absolute;
  top: 15px;
  right: 15px;
  z-index: 1;
  font-size: 16px;
}

@media (min-width:767px) {
  .c0 {
    top: 25px;
    right: 25px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled EmailButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  width: 100%;
  margin: 0 5px;
  padding: 5px 15px;
  color: rgb(255,255,255);
  margin-top: 10px;
  background-color: rgb(222,26,66);
}

.c0 svg {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg {
  fill: rgb(74,74,74);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

@media (min-width:767px) {
  .c0 {
    margin-top: 0;
  }
}

<button
  className="1 c0"
/>
`;

exports[`ButtonElements/Styled FacebookButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  width: 100%;
  margin: 0 5px;
  padding: 5px 15px;
  color: rgb(255,255,255);
  background-color: rgb(58,89,152);
}

.c0 svg {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg {
  fill: rgb(74,74,74);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled GoogleButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  width: 100%;
  margin: 0 5px;
  padding: 5px 15px;
  color: rgb(255,255,255);
  margin-top: 10px;
  background-color: rgb(216,58,44);
}

.c0 svg {
  fill: rgb(255,255,255);
}

.c0:disabled {
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0:disabled svg {
  fill: rgb(74,74,74);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

@media (min-width:767px) {
  .c0 {
    margin-top: 0;
  }
}

<button
  className="0 c0"
/>
`;

exports[`ButtonElements/Styled GreyButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
}

.c0 svg {
  fill: rgb(222,26,66);
}

.c0:hover,
.c0:focus {
  color: rgb(74,74,74);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled IconWrapperStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
  margin-right: 5px;
}

<span
  className="c0"
/>
`;

exports[`ButtonElements/Styled RedButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled RedLinkButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  border: none;
  background-color: transparent;
  display: inline-block;
  font-family: 'Roboto Bold',Arial,sans-serif;
  color: rgb(222,26,66);
  -webkit-text-decoration: underline;
  text-decoration: underline;
  margin: 0 5px;
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled SmallGreyButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(74,74,74);
  background-color: rgb(229,229,229);
  padding: 5px 15px;
}

.c0 svg {
  fill: rgb(222,26,66);
}

.c0:hover,
.c0:focus {
  color: rgb(74,74,74);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled SmallRedButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: 'Roboto Condensed Bold',Arial,serif;
  font-size: 14px;
  line-height: 18px;
  border: none;
  border-radius: 30px;
  text-transform: uppercase;
  padding: 10px 15px;
  color: rgb(255,255,255);
  background-color: rgb(222,26,66);
  padding: 5px 15px;
}

.c0 svg {
  fill: rgb(255,255,255);
}

@media (min-width:767px) {
  .c0 {
    font-size: 16px;
  }
}

@media (min-width:767px) {
  .c0 {
    padding: 10px 20px;
  }
}

<button
  className="c0"
/>
`;

exports[`ButtonElements/Styled UnstyledButtonStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  border: none;
  background-color: transparent;
}

<button
  className="c0"
/>
`;
